### PR TITLE
made changes in names of classes to fix errors

### PR DESCRIPTION
**Pull request by abhimaniac4**

### CODE REVIEW

File: library/entities/Loan.java [hunk 1]
```diff
@@ -1,79 +1,69 @@
...
+	private Date date;
+	private LoanState state;
+
+	public Loan(int loanId, Book book, Member member, Date DuE_dAtE) {
 		this.LoAn_Id = loanId;
-		this.BoOk = bOoK;
-		this.MeMbEr = mEmBeR;
+		this.book = book;
+		this.member = member;
 		this.DaTe = DuE_dAtE;
```
Comment (Lakshansm90) [inline library/entities/Loan.java]: Change the variable Name

File: library/entities/Loan.java [hunk 2]
```diff
@@ -1,79 +1,69 @@
...
-		this.StAtE = lOaN_sTaTe.CURRENT;
```
Comment (Lakshansm90) [inline library/entities/Loan.java]: Follow the naming convention rules

File: library/entities/Loan.java [hunk 3]
```diff
@@ -1,79 +1,69 @@
...
 	}
 
-	
-	public void cHeCk_OvEr_DuE() {
-		if (StAtE == lOaN_sTaTe.CURRENT &&
-			Calendar.gEtInStAnCe().gEt_DaTe().after(DaTe)) 
-			this.StAtE = lOaN_sTaTe.OVER_DUE;			
-		
+	public void checkOverDue() {
+		if (state == LoanState.CURRENT && Calendar.gEtInStAnCe().gEt_DaTe().after(DaTe))
```
Comment (Lakshansm90) [inline library/entities/Loan.java]: Use the curly brace("{}") when you define a **if**  condition

File: library/entities/Loan.java [hunk 4]
```diff
@@ -1,79 +1,69 @@
...
-	public boolean Is_OvEr_DuE() {
-		return StAtE == lOaN_sTaTe.OVER_DUE;
+	public boolean isOverDue() {
+		return state == LoanState.OVER_DUE;
 	}
 
-	
-	public Integer GeT_Id() {
+	public Integer getId() {
 		return LoAn_Id;
```
Comment (Lakshansm90) [inline library/entities/Loan.java]: check the return value variable name

File: library/payfine/PayFineControl.java [hunk 5]
```diff
@@ -50,18 +59,18 @@ public void Cancel() {
...
+			ui.display(String.format("Change: $%.2f", change));
 
 		ui.display(member.toString());
 		ui.setState(PayFineUi.UiState.COMPLETED);
 		state = ControlState.COMPLETED;
-		return ChAnGe;
+		return change;
 	}
 
 }
```
Comment (Lakshansm90) [inline library/payfine/PayFineControl.java:76]: Follow the naming convention for the below highlighted names
private Library **LiBrArY**;
public void **CardSwiped**(int memberId)
public void **Cancel**()

Use a curly brace when yu define the Condition 
if (!state.equals(ControlState.READY))
if (!state.equals(ControlState.PAYING))
if (change > 0)

**Review by Lakshansm90 — CHANGES_REQUESTED** [review]
HI @iamharisali ,
Could you please ask @abhimaniac4 to do the necessary changes and commit again.
Thanks
Lakshan

File: library/entities/Loan.java (hunk 1 above)
Comment (abhimaniac4) [reply library/entities/Loan.java]: Thanks for the feedback, I will make the changes

File: library/entities/Loan.java (hunk 2 above)
Comment (abhimaniac4) [reply library/entities/Loan.java]: Thanks for the feedback, I will make the changes

File: library/entities/Loan.java (hunk 3 above)
Comment (abhimaniac4) [reply library/entities/Loan.java]: Thanks for the feedback, I will make the changes

File: library/entities/Loan.java (hunk 4 above)
Comment (abhimaniac4) [reply library/entities/Loan.java]: Thanks for the feedback, I will make the changes

File: library/payfine/PayFineControl.java (hunk 5 above)
Comment (abhimaniac4) [reply library/payfine/PayFineControl.java:76]: Thanks for the feedback, I will make the changes

**Review by abhimaniac4 — COMMENTED** [review]
changes made @Lakshansm90

**Pull request merged**